ext/random/tests/02_engine/user_xoshiro128plusplus.phpt

```diff
@@ -1,61 +0,0 @@
---TEST--
```

Comment:
I think it would be preferable to have a test to see if PRNGs can be implemented correctly in userland.

Reply:
I thought I sent this a few days ago (I misunderstood the GH behavior)
Hope you don't mind.